packages/fxa-payments-server/src/components/PaymentMethodHeader/index.test.tsx

```diff
@@ -97,7 +97,7 @@ describe('components/PaymentMethodHeader', () => {
         MOCK_PLANS.find((p) => p.plan_id === 'plan_daily') || MOCK_PLANS[0];
       const props = { plan, onClick: () => {} };
       const expectedMsg =
-        'I authorize Mozilla, maker of Firefox products, to charge my payment method for the amount shown, according to Terms of Service and Privacy Notice, until I cancel my subscription.';
+        'I authorize Mozilla, maker of Firefox products, to charge my payment method for the amount shown, according to Terms of ServiceOpens in new window and Privacy NoticeOpens in new window, until I cancel my subscription.';
```

Comment:
Q: Is this correct? It looks odd. I didn't seen any errors in testing but should `Opens in new window` be appended to the ends of `Service` and `Notice` respectively?

Reply:
As discussed in chat, the <LinkExternal> component has this line in it, which doesn’t show up in the browser, but it does show up in textContent, which is why you get this behaviour in the test.
```
<span className="sr-only">Opens in new window</span>
```